android/fastlane/Appfile

```diff
@@ -0,0 +1,2 @@
+json_key_file("") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
```

Comment:
Strange, seems like we don't even have this in integreat.

Reply:
This should be removed and directly passed via environment variables as base64 in the fastfile

Reply:
done